### PR TITLE
feat(footer): a11y audit controller

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did

1. move footer html a11y audit controller from #814 to here
2. refactor

## Notes to Reviewers
This may be closed in favour of #813

### CODE REVIEW

File: elements/rh-footer/rh-footer-html-audit-controller.ts
Comment (nikkimk) [inline elements/rh-footer/rh-footer-html-audit-controller.ts:1]: Question from @zeroedin: Is this something we'll want to use on subnav and other nav components with links?

File: elements/rh-footer/RhFooter.ts
Comment (bennypowers) [inline elements/rh-footer/RhFooter.ts:1]: let's wait on merging this

**Comment by markcaron** [comment]
@nikkimk and @bennypowers do either of you mind revisiting this? Wondering where we're at on it.

**Comment by zeroedin** [comment]
@bennypowers @markcaron Should we close this PR?  Seems like the idea is no longer something we are pursuing or warranted at this time.

**Comment by bennypowers** [comment]
We'll address this with selector hound

**Pull request closed without merging**